.docs/01-introduction.md

```diff
@@ -0,0 +1,10 @@
+[:book:][index] - [Getting Started →][next]
+
+# Introduction
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur eget commodo magna. Nam varius, purus tincidunt pulvinar auctor, sapien lorem consequat tortor, in cursus enim sapien vel erat. Donec non sollicitudin neque. Integer rutrum egestas purus ac porttitor. Morbi maximus metus dapibus iaculis molestie. Nam euismod sed est quis auctor. Aenean at odio vitae orci sodales lobortis nec a nibh. Maecenas condimentum ipsum risus, eu pellentesque augue facilisis a. In hac habitasse platea dictumst. Quisque fermentum accumsan felis, eu venenatis nisl pulvinar vel. Nam eu accumsan est. Sed viverra vel ligula sit amet placerat. Donec molestie ut mauris nec egestas.
```

Comment:
A thorough overview would cover:

- JSON, Protobuf, Thrift: Encodings
- HTTP/1, gRPC, TChannel: Transports
- Caller, Service, Procedure, Headers: Metadata
- Baggage, Tracing: Context
- Round Robin, Least Pending: Peer Lists
- Peer List Bindings
- Unary, Oneway, Streaming: RPC Types
- Configuration and Plugins

But perhaps we should make this a light introduction and have a separate "Concepts" section as reference material.